resources/views/platform/completion-progress-page.blade.php

```diff
@@ -13,12 +13,10 @@
     $headingLabel = $targetUsername . "'s Completion Progress";
 }
 
-// TODO: Once using denormalized data, come up with a good page description.
-// Doesn't matter right now because these pages don't generate any SEO juice.
-// $pageDescription = "";
+$pageDescription = "View {$targetUsername}'s game completion stats and milestones on RetroAchievements. Track their played, unfinished, and mastered games from various systems.";
```

Comment:
Do we need to worry about the crawler following the filter links (Played, Unfinished, Beaten, Completed, Mastered)?

Reply:
Good call; I got nervous at first but it looks like Laravel handles this for us by setting a canonical URL for the page:

![Screenshot 2023-11-17 at 6 07 41 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/94ec9690-99ce-4255-902c-46e2bce625fc)

This is correctly set regardless of what filters are applied, so crawlers will not see filtered content as being duplicative.